File: Code/cpp/division.cpp
#include<iostream>
using namespace std;
int main()
{
    int num1, num2;

    cout << "Enter Dividend: ";
    cin >> num1;
    cout << "Enter Divisor: ";
    cin >> num2;

    cout << "Quotient: " << num1 / num2;
    cout << "\nRemainder: " << num1 % num2;
}

File: Code/java/division.java
import java.util.Scanner;

class Division
{
    public static void main(String[] args)
    {
        int num1, num2;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Dividend: ");
        num1 = scanner.nextInt();
        System.out.println("Enter Divisor: ");
        num2 = scanner.nextInt();

        System.out.println("Quotient: " + num1 / num2);
        System.out.println("Remainder: " + num1 % num2);

        scanner.close();
    }
}